# defines a YOLO detection result

# class probability
int32 class_id
string class_name
float64 score

# 2D bounding box surrounding the object in pixels
yolo_ros2_following/BoundingBox2D bbox

